photon-lib/src/main/scala/com/linkedin/photon/ml/util/VectorUtils.scala

```diff
@@ -14,14 +14,14 @@
  */
 package com.linkedin.photon.ml.util
 
-import scala.collection.mutable
-
 import breeze.linalg.{DenseVector, SparseVector, Vector}
-import org.apache.spark.ml.linalg.{Vector => SparkMLVector}
-import org.apache.spark.mllib.linalg.{Vectors, DenseVector => SparkDenseVector, SparseVector => SparkSparseVector, Vector => SparkVector}
+import org.apache.spark.ml.linalg.{DenseVector => SparkMLDenseVector, SparseVector => SparkMLSparseVector, Vector => SparkMLVector}
+import org.apache.spark.mllib.linalg.{DenseVector => SparkDenseVector, SparseVector => SparkSparseVector, Vector => SparkVector}
+
+import scala.collection.mutable
```

Comment:
Please move this import to it's original location - this is the only style issue for the PR

Reply:
reverted.

Reply:
sorry for the miss.